packages/@sanity/base/sanity.json

```diff
@@ -368,6 +358,10 @@
       "name": "part:@sanity/base/datastore/document",
       "path": "datastores/document"
     },
+    {
```

Comment:
We should potentially start to think about reducing the number of parts we expose - for instance by exposing a `@sanity/base/datastores` directly so you could import from that directly? This would allow for better autocomplete in editors, for instance. Not _critical_ right now, but worth thinking about

Reply:
Good idea, should probably be done in regard to the refactoring of datastores as I mentioned in your first comment.